spec/serializers/api/participant_serializer_spec.rb

```diff
@@ -98,6 +98,14 @@
           }.stringify_keys,
         ])
       end
+
+      context "when application is not accepted" do
+        let(:application) { create(:application, lead_provider:) }
```

Comment:
could this test instead check for the applicant having multiple applications for the same provider, but not accepted? same below